XMLDocGen/Tree/Models/NodeFunction.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace XMLDocGen.Tree.Models
+{
+    class NodeFunction : INode
+    {
+        public List<INode> Child { get; set; } = new List<INode>();
+
+        public EType Type { get => EType.Function; }
+
+        public string Name { get; }
+
+        public string Description { get; }
+
+        public UInt16 Args { get; }
+
+        public string Returning { get; }
+
+        public NodeFunction(string name, string desc, UInt16 args, string ret)
```

Comment:
args is count of function\method arguments or it's your mistake and it's must be list of arguments

Reply:
See https://github.com/Nelyndir/XDCtoMarkdown/blob/a862176101bf7f2ed90a7ca0d37ab825994ac9f2/XMLDocGen/Models/Method.cs#L22